src/Mono.Android/Xamarin.Android.Net/AndroidClientHandler.Legacy.cs

```diff
@@ -56,6 +56,7 @@ namespace Xamarin.Android.Net
 	/// if they use a certificate with a fully validated trust chain) unless you store the CA certificates from your Android system in <see cref="TrustedCerts"/> along with
 	/// the self-signed certificate(s).</para>
 	/// </remarks>
+	[Obsolete("AndroidClientHandler has been deprecated. Use AndroidMessageHandler instead.")]
 	public class AndroidClientHandler : HttpClientHandler
```

Comment:
Should this be obsolete only in .NET 6? Or Xamarin.Android as well?

Reply:
I removed the attribute from the Xamarin.Android version